week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,7 +24,7 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  const isBigEnough = num + 5;
```

Comment:
sorry for this being vague. So usually when you read a variable name that asks a question like isItLarger, isItBig, isInfoRight, usually means it's a boolean.
Your work is correct. To follow the above-mentioned practice
```isBigEnough=num>=10```